polygon/sync/block_downloader.go

```diff
@@ -162,7 +166,14 @@ func (d *blockDownloader) downloadBlocksUsingWaypoints(ctx context.Context, wayp
 				"kind", reflect.TypeOf(waypointsBatch[0]),
 				"peerCount", len(peers),
 				"maxWorkers", d.maxWorkers,
+				"blk/s", fmt.Sprintf("%.2f", float64(sum)/time.Since(logStartTime).Seconds()),
+				"bytes/s", fmt.Sprintf("%s", common.ByteCount(uint64(totalSize/time.Since(logStartTime).Seconds()))),
```

Comment:
please add a helper function to calculate speed, and check for zero division there

Reply:
I think we are safe without this check, since the log interval is set to 30 seconds. We similar divisions elsewhere in the codebase without checks
https://github.com/ledgerwatch/erigon/blob/bd378f2533052e1a97f61b2ad4cb855d6e385602/cl/phase1/stages/stage_history_download.go#L303